mcs/class/Makefile

```diff
@@ -300,6 +298,11 @@ xbuild_4_0_dirs := \
 	$(xbuild_2_0_dirs)	\
 	Microsoft.Build
 
+ifdef HAVE_BTLS
```

Comment:
Hmm, this doesn't seem to work, it'll still compile it just with a different name (which won't work since it's missing the dependencies from System.dll when BTLS is disabled)